src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Common/SyntaxNodeExtensions.cs

```diff
@@ -4,15 +4,38 @@
 using System;
 using System.Linq;
 using Microsoft.CodeAnalysis;
+using Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Common;
+
 using CS = Microsoft.CodeAnalysis.CSharp;
 using CSSyntax = Microsoft.CodeAnalysis.CSharp.Syntax;
+
 using VB = Microsoft.CodeAnalysis.VisualBasic;
 using VBSyntax = Microsoft.CodeAnalysis.VisualBasic.Syntax;
 
 namespace Microsoft.DotNet.UpgradeAssistant.Extensions.Default
 {
     public static class SyntaxNodeExtensions
     {
+        public static bool IsVisualBasic(this SyntaxNode node) => node?.Language == LanguageNames.VisualBasic;
+
+        public static bool IsCSharp(this SyntaxNode node) => node?.Language == LanguageNames.CSharp;
+
+        public static SyntaxNode AddArgumentToInvocation(this SyntaxNode invocationNode, SyntaxNode argument)
```

Comment:
this looks nice, let's merge this PR first so I can reuse this in the PR for binaryFormatter